distribution/alliance/pom.xml

```diff
@@ -0,0 +1,442 @@
+                                </artifactItem>
+                                <artifactItem>
+                                    <groupId>ddf.platform.security</groupId>
+                                    <artifactId>docs</artifactId>
+                                    <version>${ddf.version}</version>
+                                    <outputDirectory>${setup.folder}/documentation/security</outputDirectory>
+                                    <classifier>export</classifier>
+                                    <type>zip</type>
+                                </artifactItem>
+                                <artifactItem>
+                                    <groupId>org.codice.ddf.spatial</groupId>
+                                    <artifactId>docs</artifactId>
+                                    <version>${ddf.version}</version>
+                                    <outputDirectory>${setup.folder}/documentation/spatial</outputDirectory>
+                                    <classifier>export</classifier>
+                                    <type>zip</type>
+                                </artifactItem>
+                                <artifactItem>
+                                    <groupId>ddf.ui</groupId>
+                                    <artifactId>docs</artifactId>
+                                    <version>${ddf.version}</version>
+                                    <outputDirectory>${setup.folder}/documentation/ui</outputDirectory>
+                                    <classifier>export</classifier>
+                                    <type>zip</type>
+                                </artifactItem>
+                            </artifactItems>
+                            <overWriteSnapshots>true</overWriteSnapshots>
+                        </configuration>
+                    </execution>
+                </executions>
+            </plugin>
+            <!-- Overwrite system.properties settings -->
+            <plugin>
+                <groupId>com.google.code.maven-replacer-plugin</groupId>
+                <artifactId>replacer</artifactId>
+                <version>1.5.3</version>
+                <executions>
+                    <execution>
+                        <phase>generate-resources</phase>
+                        <goals>
+                            <goal>replace</goal>
+                        </goals>
+                    </execution>
+                </executions>
+                <configuration>
+                    <file>${project.basedir}/target/dependencies/ddf-kernel-${ddf.version}/etc/system.properties</file>
+                    <replacements>
+                        <replacement>
+                            <token>ddf.distribution</token>
+                            <value>Alliance Distribution</value>
+                        </replacement>
+                        <replacement>
+                            <token>${ddf.version}</token>
```

Comment:
Feels like we should have a more strict token match - otherwise it's just going to replace all "2.9.0" which seems like it could be risky.